openstack/css/v1/cluster/requests.go

```diff
@@ -0,0 +1,288 @@
+	// Subnet ID. All instances in a cluster must have the same subnets and security groups.
+	NetId string `json:"netId" required:"true"`
+	// Security group ID. All instances in a cluster must have the same subnets and security groups.
+	SecurityGroupId string `json:"securityGroupId" required:"true"`
+	// VPC ID, which is used for configuring cluster network.
+	VpcId string `json:"vpcId" required:"true"`
+}
+
+// CreateClusterInstanceVolumeBody volume
+type CreateClusterInstanceVolumeBody struct {
+	// Volume size, which must be a multiple of 4 and 10.  Unit: GB
+	Size int `json:"size" required:"true"`
+	// COMMON: Common I/O
+	// HIGH: High I/O
+	// ULTRAHIGH: Ultra-high I/O
+	VolumeType string `json:"volume_type" required:"true"`
+}
+
+type CreateClusterEncryptionBody struct {
+	// Value 1 indicates encryption is performed, and value 0 indicates encryption is not performed.
+	SystemEncrypted string `json:"systemEncrypted" required:"true"`
+	// Key ID.
+	// The Default Master Keys cannot be used to create grants. Specifically, you cannot use Default Master Keys whose
+	// aliases end with /default in KMS to create clusters.
+	// After a cluster is created, do not delete the key used by the cluster. Otherwise,
+	// the cluster will become unavailable.
+	SystemCmkid string `json:"systemCmkid" required:"true"`
+}
+
+type ListClustersDetailsOpts struct {
+	// Start value of the query. The default value is 1, indicating that the query starts from the first cluster.
+	Start int `q:"start"`
+	// Number of clusters to be queried. The default value is 10, indicating that 10 clusters are queried at a time.
+	Limit int `q:"limit"`
+}
+
+type RoleExtendReq struct {
+	// Detailed description about the cluster scale-out request. For detai
+	Grow []RoleExtendGrowReq `json:"grow" required:"true"`
+}
+
+// RoleExtendGrowReq
+type RoleExtendGrowReq struct {
+	// Storage capacity of the instance to be expanded. The total storage capacity of existing instances
+	// and newly added instances in a cluster cannot exceed the maximum instance storage capacity allowed when a
+	// cluster is being created. In addition, you can expand the instance storage capacity for a cluster
+	// for up to six times.Unit: GB
+	Disksize *int `json:"disksize" required:"true"`
+	// Number of instances to be scaled out. The total number of existing instances and newly added instances
+	// in a cluster cannot exceed 32.
+	Nodesize *int `json:"nodesize" required:"true"`
+	// Type of the instance to be scaled out. Select at least one from ess, ess-cold, ess-master, and ess-client.
+	// You can only add instances rather than increase storage capacity on nodes of the ess-master and ess-client types.
+	Type string `json:"type" required:"true"`
+}
+
+var RequestOpts = golangsdk.RequestOpts{
+	MoreHeaders: map[string]string{"Content-Type": "application/json", "X-Language": "en-us"},
+}
+
+func CreateCluster(c *golangsdk.ServiceClient, opts CreateClusterReq) (*CreateClusterResponse, error) {
```

Comment:
as the package name is *cluster*, suggest the name of functions and structs does not contain cluster.
for this function, just rename it to Create. You can refer to the code style from other packages.

Reply:
ok, done